library/build.gradle

```diff
@@ -8,6 +19,7 @@ android {
         minSdkVersion 17
         targetSdkVersion 23
         versionCode 1
+        // NOTE: When updating versionName, be sure to update version in publishing group below too
```

Comment:
I'm going to try using a variable one more time before I give up and just merge this in.